README.md

```diff
@@ -40,10 +40,7 @@ targets and mediums.
 
    source "${bashutils_path}"
 
-   fetchFromGithub "facebookincubator" "TTPForge" "v1.0.3" ttpforge
-
-   # Optionally, if you are using the `gh` cli:
-   fetchFromGithub "facebookincubator" "TTPForge" "v1.0.3" ttpforge $GITHUB_TOKEN
+   fetchFromGithub "facebookincubator" "TTPForge" "v1.0.8" ttpforge
```

Comment:
This is how it would look if we want to default to latest vs. version pinning in the instructions:

```suggestion
   fetchFromGithub "facebookincubator" "TTPForge" "latest" ttpforge
```

Reply:
Thank you - good change